index.js

```diff
@@ -0,0 +1,24 @@
+import sass from "sass";
+import { promisify } from "util";
+import { writeFile } from "fs";
+const sassRenderPromise = promisify(sass.render);
```

Comment:
the Sass docs actually claim that using the sync render function is 2x as fast (although I'm sure we're splitting hairs for a build process) — so maybe we don't need to promisify it either.

https://github.com/sass/dart-sass#javascript-api